src/odemis/gui/cont/tabs.py

```diff
@@ -476,13 +472,17 @@ def __init__(self, name, button, panel, main_frame, main_data):
             sem_stream_cont = self._streambar_controller.addStream(sem_stream, add_to_view=True)
             sem_stream_cont.stream_panel.show_remove_btn(False)
 
+        # Only enable the tab when the stage is at the right position
         if self.main_data.role == "enzel":
             self.stage = self.tab_data_model.main.stage
+            self._allowed_targets = [THREE_BEAMS, SEM_IMAGING]
```

Comment:
Don't you think we should also allow this when we are in ALIGNMENT?
It is a really small chance that someone wants to do the factory setting and go in the localization tab but it is not dangerous or anything.

Reply:
Indeed. I think that in the "ideal" version the ALIGNMENT position shouldn't be a reported position, but just a "specific position" within the THREE_BEAMS posture. However, for now that's not how it works. I'll add ALIGNMENT.